src/job.py

```diff
@@ -19,11 +20,25 @@ def __init__(self, est, lst, load_profile):
     def __eq__(self, other):
         return self.est == other.est and \
                self.lst == other.lst and \
-               self.load_profile == other.load_profile
+               self.load_profile.equals(other.load_profile)
 
     def to_message(self):
         return {
             'est': self.est,
             'lst': self.lst,
             'load_profile': self.load_profile
         }
+
+    def power(self, t):

```

Comment:
function yielding accumulated power in time _t_. If `t<0`, it returns 0. If _t_ is in the interval of the load profile, it returns the interpolated value (see the test below to get an idea of the function's behavior). If _t_ is beyond the load profile's interval, the function returns the accumulated power at the end of the interval.